### PR TITLE
Packaging and Test Improvements

### PR DESCRIPTION
**Pull request by Teque5**
- package no longer creates a dozen random sigmf files when calling `pytest`
- re-wrote Archive tests
- will fix project description and insert `README.md` into [our pypi page](https://pypi.org/project/SigMF/) once I upload new version
- incremented to `v1.1.3`

### CODE REVIEW

File: sigmf/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
-__version__ = '1.1.2'
+__version__ = "1.1.3"
```
Comment (gmabey) [inline sigmf/__init__.py:7]: This is a double increment -- current version is 1.1.1

File: tests/conftest.py [hunk 2]
```diff
@@ -29,13 +29,15 @@
 
 @pytest.fixture
 def test_data_file():
+    """when called, yeilds temporary file"""
```
Comment (gmabey) [inline tests/conftest.py:32]: yeilds -> yields

**Review by gmabey — APPROVED** [review]
Looks great overall! I am curious why you seem to prefer double-quotes for strings over single quotes though.

File: sigmf/__init__.py (hunk 1 above)
Comment (Teque5) [reply sigmf/__init__.py:7]: I pushed v1.1.2 to pypi yesterday.

File: tests/conftest.py (hunk 2 above)
Comment (Teque5) [reply tests/conftest.py:32]: FYI If you have a single change like this you can [use the suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) — I just click **OK** and it gets committed. That way I can avoid opening an editor and just do it on my phone.
```suggestion
    """when called, yields temporary file"""
```

**Comment by Teque5** [comment]
> Looks great overall! I am curious why you seem to prefer double-quotes for strings over single quotes though.

I actually don't, but instead of manually fixing things with `flake8`, I've been using [black](https://github.com/psf/black) lately that just forces PEP8 (with double quotes). Also lately I've been coding in Rust and it's a good reminder to me that over there strings are "like this" and unicode single characters are '✪'.

**Pull request merged**